contributing_to_docs/doc_guidelines.adoc

```diff
@@ -259,6 +290,25 @@ link:https://redhat-documentation.github.io/modular-docs/#creating-procedure-mod
 When needed, use `.Prerequisites`, `.Next steps`, or `.Additional resources` syntax to suppress TOC formatting within a module. Do not use `==` syntax for these headings in modules. Because you cannot use the xrefs in modules, if you need to include a link under one of these headings, place the entire subsection in the assembly instead.
 ====
 
+[id="writing-text-snippets"]
+== Writing text snippets
+A _text snippet_ is an optional component that lets you reuse content in multiple modules and assemblies. Text snippets are not a substitute for modules, but instead are a more granular form of content reuse. While a module is content that a reader can understand on its own (article) or as part of a larger body of work (assembly), a text snippet is not be published or cross referenced on its own.
```

Comment:
Maybe a little bit more wording in these parentheticals to help clarify the meaning? Particularly since we haven't used the term "article" until this point.

> ... on its own (like an article) or as part of a larger body of work (like in an assembly)